main.go

```diff
@@ -17,6 +18,8 @@ func main() {
 		Behaviors: behaviors,
 	}
 
```

Comment:
Please print "Waiting 1 second for test clients to become ready..." and then "Beginning matrix test"